pcsx2/GS/Renderers/DX11/GSDevice11.cpp

```diff
@@ -798,6 +794,24 @@ void GSDevice11::UpdateCLUTTexture(GSTexture* sTex, u32 offsetX, u32 offsetY, GS
 	StretchRect(sTex, GSVector4::zero(), dTex, dRect, m_convert.ps[static_cast<int>(shader)].get(), m_merge.cb.get(), nullptr, false);
 }
 
+void GSDevice11::ConvertToIndexedTexture(GSTexture* sTex, u32 offsetX, u32 offsetY, u32 SBW, u32 SPSM, GSTexture* dTex, u32 DBW, u32 DPSM)
```

Comment:
The offsetX and offsetY parameters are not used in the function. It is not clear if this is intentional, but if they are not used, they should be removed to avoid confusion.